kubernetes/platform-services/Chart.yaml

```diff
@@ -76,3 +76,7 @@ dependencies:
     repository: "https://storage.googleapis.com/hypertrace-helm-charts"
     version: 0.1.7
     condition: kafka-topic-creator.enabled
+  - name: hypertrace-alert-engine
+    repository: "https://storage.googleapis.com/hypertrace-helm-charts"
+    version: 0.1.4
+    condition: alerting.enabled 
```

Comment:
should we disable it by default?

Reply:
The reason being, there won't be any slack URL and all so this will be failing anyways.

Reply:
it is disabled by default

Reply:
cool.